addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/handler/WebserviceHandler.java

```diff
@@ -66,7 +69,19 @@ public WebserviceHandler(Bridge bridge) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        // no code needed.
+        try {
+            if (channelUID.getId().contentEquals(CHANNEL_HOLIDAY)) {
+                logger.debug("HandleCommand recieved: {}", channelUID.getId());
+                if (command.toString() == "OFF") {
+                    isHoliday = false;
```

Comment:
I'm trying to use the "OnOffType", however the IDE asks for an import of "org.openhab.core.library.types".
This time I checked ALL the comments I got so far and found a remark of Kai stating "..bindings should not have any dependencies to openHAB core, only to ESH APIs. ".
Why doesn't the IDE give the suggestion to import "org.eclipse.smarthome.core.library.types.OnOffType" instead of only the "org.openhab.core....." ones?